src/components/WeekDaysSelector.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <h5>Avalable Weeks</h5>
+    <div class="week">
+      <div class="days" v-for="weekDay in weekDays" :key="weekDay.id">
```

Comment:
Please `concat` a unique string with `id` to make it true unique in our dom.

Reply:
Done